<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PackagesPage</class>
 <widget class="QWidget" name="PackagesPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="listWidget"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="buttonAdd">
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset resource="grauchy.qrc">
         <normaloff>:/resources/edit_add.png</normaloff>:/resources/edit_add.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonEdit">
       <property name="text">
        <string>&amp;Edit</string>
       </property>
       <property name="icon">
        <iconset resource="grauchy.qrc">
         <normaloff>:/resources/edit_view.png</normaloff>:/resources/edit_view.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonRemove">
       <property name="text">
        <string>&amp;Remove</string>
       </property>
       <property name="icon">
        <iconset resource="grauchy.qrc">
         <normaloff>:/resources/edit_remove.png</normaloff>:/resources/edit_remove.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="grauchy.qrc"/>
 </resources>
 <connections/>
</ui>
